doc/tutorial/basic/tutorial.rst

```diff
@@ -136,11 +136,30 @@ learn::
     <sphx_glr_auto_examples_classification_plot_digits_classification.py>` illustrates how starting
     from the original problem one can shape the data for consumption in
     scikit-learn.
-    
+
 .. topic:: Loading from external datasets
 
     To load from an external dataset, please refer to :ref:`loading external datasets <external_datasets>`.
 
+.. topic:: Accessing source files
```

Comment:
ack.

BTW, people have to load_ dataset first to see the path to data files. Looks a bit strange. Do you think is it expected behaviour?